e2e-pw/src/oss/fixtures/loader.ts

```diff
@@ -145,8 +150,11 @@ export class OssLoader extends AbstractFiftyoneLoader {
       }
     }
 
-    await page.waitForSelector("[data-cy=flashlight-section]", {
-      state: "visible",
-    });
+    await page.waitForSelector(
```

Comment:
nit: I think you can use page.getByTestId('TEST_ID').waitFor({ state: 'visible'}) - not sure if it makes a difference

Reply:
Right, I saw that too. I hesitated because I thought @sashankaryal had a reason for this approach

Reply:
Appreciate the benefit of the doubt, Ben!
Let's just say I was learning Playwright. Feel free to change both of these to `getByTestId::waitFor`